WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -319,4 +318,20 @@ private extension CartView {
     return CartView()
         .environment(posModel)
 }
+
+@available(iOS 17.0, *)
+#Preview("Cart with one item") {
+    let itemsController = PointOfSalePreviewItemsController()
```

Comment:
Not needed!
```suggestion
```

Reply:
Removed in https://github.com/woocommerce/woocommerce-ios/pull/15089/commits/c90aa59dfc2ebbb236f7cfcc00b471619cfae132.